akka-http/src/main/resources/reference.conf

```diff
@@ -39,5 +39,5 @@ akka.http.routing {
 
   # Fully qualified config path which holds the dispatcher configuration
   # to be used by FlowMaterialiser when creating Actors for IO operations.
-  file-io-dispatcher = ${akka.io.tcp.file-io-dispatcher}
+  file-io-dispatcher = ${akka.io.tcp.file-io-dispatcher} // TODO point to akka.stream.file-io-dispatcher instead?
```

Comment:
Good question. I would say fixed size is likely better since the contended resource is not the CPU, therefore core count is less significant (I wouldn't go as far as to say insignificant though)